first_set_of_refactorings/extract_variable/example.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+# formerly: Introduce Explaining Variables
+# inverse of: Inline Variable
+# @see https://memberservices.informit.com/my_account/webedition/9780135425664/html/extractfunction.html
+
+# Goal: Add a name to an expression
+
+module FirstSetOfRefactorings
+  module ExtractVaiable
+    module Example
```

Comment:
I've been dropping the word "Example" so this would be more like `WithoutAClass`